bridge/ffi/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "graph-sdk-ffi"
```

Comment:
Can we use consistent naming across our GraphSDK crates? Other crates are `dsnp-graph-*`; how about `dsnp-graph-ffi`?